femmt/component.py

```diff
@@ -1772,6 +2083,411 @@ def calculate_and_write_log(self, sweep_number: int = 1, currents: List = None,
         with open(self.file_data.e_m_results_log_path, "w+", encoding='utf-8') as outfile:
             json.dump(log_dict, outfile, indent=2, ensure_ascii=False)
 
+    def calculate_and_write_log_try(self):
```

Comment:
what is the _try for?

Reply:
it is not clear what is the difference between the sub-functions what can be found into this function. A small summary would be nice into the first doc-string